### DISCUSSION TITLE
Bit like struct

### DISCUSSION THREAD

**Discussion by jbbjarnason** [stephenberry/glaze#1051] [Q&A] [answered]
Was there any  explicit support for bit like struct, e.g.: 
```cpp 
struct {
  bool no_reply_expected : 1 { false };
  bool no_auto_start : 1 { false };
  bool allow_interactive_authorization : 1 { false };
  std::uint8_t reserved : 5 { 0 };
};
```

Or would I just use glz::custom ?

https://gcc.godbolt.org/z/ve3z69K5h

**Comment by stephenberry** [comment, marked as answer]
Bit fields are tricky because they don't support references or member function pointers. It's best to do bitwise logic in glz::custom.
